.github/workflows/publish-helm-chart.yml

```
@@ -0,0 +1,90 @@
+name: Publish Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-helm-list:
```

```suggestion
  build-chart-list:
```

Helm is to charts as Docker is to images. Please also update all the other references + vars